cmd/zq/zq.go

```diff
@@ -193,11 +194,20 @@ func (c *Command) loadFile(path string) (zson.Reader, error) {
 	if info.IsDir() {
 		return nil, errInvalidFile("is a directory")
 	}
+	// XXX this should go away soon once we have functionality to peek at the
+	// stream.
+	var reader string
+	switch ext := filepath.Ext(path); ext {
+	default:
```

Comment:
Nit: I think it'd be nice to be consistent about where we put `default`, and with one exception in this repository, it follows the final `case`.